definition-src/Game/LambdaHack/Content/RuleKind.hs

```diff
@@ -35,6 +35,24 @@ data RuleContent = RuleContent
   , rstairWordCarried :: [Text]    -- ^ words that can't be dropped from stair
                                    --   name as it goes through levels
   , rsymbolProjectile :: Char
+  , rsymbolLight      :: Char
+  , rsymbolTool       :: Char
+  , rsymbolSpecial    :: Char  -- don't overuse, because it clashes with projectiles
```

Comment:
These comments are mostly about a particular configuration of symbols, not of the API (in particular, some comments differ in Allure). So I'd keep them only in a concrete game content, not this module, which is the game content API.